config/stage_test/stage_test.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: stage-test
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: stage-test-service-account
+      containers:
+        - image: quay.io/cloudservices/floorist-operator:latest
```

Comment:
This should not be hardcoded to this image as it needs to be variable.

Use:
```
controller:latest
```
It is later changed by Kustomization. Like here https://github.com/RedHatInsights/floorist-operator/blob/063d73284e77117252d812e512c7a7501d7adcd1/config/manager/kustomization.yaml#L14-L17

The configuration is changing by `make deploy` target (https://github.com/RedHatInsights/floorist-operator/blob/063d73284e77117252d812e512c7a7501d7adcd1/Makefile#L112), controlled by `IMAGE_TAG_BASE`  and `VERSION` env variables. 

Add such config to `config/stage_test/kustomization.yaml` and extend the `deploy` target, or create a new one for deploying stage test.